### DISCUSSION TITLE
What's the procedure to install Control Net to Comfy UI?

### DISCUSSION THREAD

**Discussion by Honzomatic** [comfyanonymous/ComfyUI#1212] [General]
It's all in the title. I find lots of reference for installing to Invoke or to Stable Diffusion itself, but Comfy doesn't seem to want to show me the Stable Diffusion front end. How do I proceed?

**Comment by Lorg0n** [comment]
Download the model, drop it into the folder ./ComfyUI/models/controlnet/
Then you build something like this
![image](https://github.com/comfyanonymous/ComfyUI/assets/37798530/dfc3b3c5-8622-4ff3-8f78-c3f705eb4c32)

**Comment by Honzomatic** [comment]
Thank you! Where do I get it? I see methods for downloading controlnet from the extensions tab of Stable Diffusion, but even though I have it installed via Comfy UI, I don't seem to be able to access Stable Diffusion itself. What am I doing wrong?

**Comment by Honzomatic** [comment]
Found it!

**Reply by titlestad** [reply]
Can you please post the link? You ask for help, once you figure it out, you don't leave a trace :-D - it would be helpful for others who are looking for the same thing. :pray:

**Reply by phineas-pta** [reply]
https://comfyanonymous.github.io/ComfyUI_examples/controlnet/
